alita_sdk/runtime/langchain/document_loaders/constants.py

```diff
@@ -139,7 +139,7 @@ class LoaderProperties(Enum):
     '.pdf': {
         'class': AlitaPDFLoader,
         'is_multimodal_processing': False,
-        'kwargs': {},
+        'kwargs': {LoaderProperties.LLM.value: None},
```

Comment:
It is expected to be empty.
Revert please.
The final state of config can be viewed in debug here https://github.com/ProjectAlita/alita-sdk/blob/4d62403083226f0d0c7cdef9c4537b17e69c7288/alita_sdk/tools/utils/content_parser.py#L240 in loader_kwargs variable